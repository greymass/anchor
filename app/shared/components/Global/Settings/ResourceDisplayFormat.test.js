import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GlobalSettingsResourceDisplayFormat from './ResourceDisplayFormat';

const mockProps = {
  defaultValue: 'defaultValue',
  name: 'resourceDisplayFormat',
  selection: [],
  t: text => text,
};

describe('GlobalSettingsResourceDisplayFormat', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GlobalSettingsResourceDisplayFormat {...mockProps} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
