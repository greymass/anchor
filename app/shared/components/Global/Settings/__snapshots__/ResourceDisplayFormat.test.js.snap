// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalSettingsResourceDisplayFormat renders correctly 1`] = `
<I18n
  bindI18n="languageChanged loaded"
  bindStore="added removed"
  defaultValue="defaultValue"
  name="resourceDisplayFormat"
  ns={
    Array [
      "global",
    ]
  }
  nsMode="default"
  omitBoundRerender={true}
  selection={Array []}
  t={[Function]}
  translateFuncName="t"
  usePureComponent={false}
  wait={false}
  withRef={false}
>
  <Component />
</I18n>
`;
